### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Naktakala**
# Minor changes

## Framework
- Added an error to Exodus exporting when trying to export polygons or polyhedrons.
- Upgraded some error messages for wavefront.

## LBS
- Diffusion MIP can now optionally use the output vector as an initial guess.
- Renamed `WGDSAProjectBackPhi0` to `GSProjectBackPhi0`
- Renamed `WGDSACopyOnlyPhi0` to `WGSCopyOnlyPhi0`
- Added Transport-Ops and Diffusion-Ops counters to solvers
- Cleaned up `PowerIterationKEigen1` 
- Added 4 free power iterations to `PowerIterationKEigen2`
- Refreshed LaTeX document for KEigen stuff

### CODE REVIEW

File: modules/LinearBoltzmannSolvers/A_LBSSolver/Acceleration/nl_keigen_acc_residual_func.cc [hunk 1]
```diff
@@ -59,12 +59,12 @@ PetscErrorCode
     (const VecDbl& input)
   {
     chi_math::Set(phi_temp, 0.0);
-    lbs_solver.WGDSAProjectBackPhi0(front_gs, /*in*/input,
+    lbs_solver.GSProjectBackPhi0(front_gs, /*in*/input,
```
Comment (ragusa) [inline modules/LinearBoltzmannSolvers/A_LBSSolver/Acceleration/nl_keigen_acc_residual_func.cc:62]: The old name was WGDSA... and now it is GS... What is GS for? Group-set??

File: modules/LinearBoltzmannSolvers/A_LBSSolver/Acceleration/nl_keigen_acc_residual_func.cc [hunk 2]
```diff
@@ -59,12 +59,12 @@ PetscErrorCode
...
       /*out*/phi_temp);
 
     SetLBSFissionSource(/*input*/phi_temp, /*output*/q_moments_local);
 
-    auto output = lbs_solver.WGDSACopyOnlyPhi0(front_gs, q_moments_local);
+    auto output = lbs_solver.WGSCopyOnlyPhi0(front_gs, q_moments_local);
```
Comment (ragusa) [inline modules/LinearBoltzmannSolvers/A_LBSSolver/Acceleration/nl_keigen_acc_residual_func.cc:67]: old name: WGDSA... (witnhin group DSA, right?) new name is WGS which may mean within group set, right?

**Review by ragusa — APPROVED** [review]
good. just a couple of Q that you can reply to via email, for my own education.

**Review by zachhardy — APPROVED** [review]

File: modules/LinearBoltzmannSolvers/A_LBSSolver/Acceleration/nl_keigen_acc_residual_func.cc (hunk 2 above)
Comment (Naktakala) [reply modules/LinearBoltzmannSolvers/A_LBSSolver/Acceleration/nl_keigen_acc_residual_func.cc:67]: I've always had trouble with this. PDT called it WGSDSA which could mean 2 things: i) It applied DSA to all the groups within the Groupset. ii) It also did only within-group acceleration.

**Pull request merged**